src/config/build_config.py

```diff
@@ -1,13 +1,16 @@
+from . import parse_dlc_developer_configs
+
 # Please only set it to True if you are preparing an EI related PR
 # Do remember to revert it back to False before merging any PR (including EI dedicated PR)
-ENABLE_EI_MODE = False
+ENABLE_EI_MODE = parse_dlc_developer_configs("dev", "ei_mode")
```

Comment:
Eventually we are adding a wrapper for the temp configs?

Reply:
eventually, yeah we can remove this layer of abstraction